kratos/utilities/delaunator_utilities.cpp

```diff
@@ -0,0 +1,207 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+#define REAL double
+
+#pragma once
```

Comment:
this is not supported by all compilers, can you please do what we usually do

Reply:
I did what we usually do before, i was confused with @RiccardoRossi comment